src/index.tsx

```diff
@@ -1,28 +1,58 @@
 import { NitroModules } from 'react-native-nitro-modules';
+import { Platform } from 'react-native';
 
 import type {
   DeclaredAgeRangeResult,
   PlayAgeRangeDeclaration,
   PlayAgeRangeDeclarationResult,
 } from './PlayAgeRangeDeclaration.nitro';
 
+import { ageRangeThresholdManager } from './AgeRangeThresholdManager';
+
+import {
+  getIsConsideredOlderThaniOS,
+  getIsConsideredOlderThanAndroid,
+} from './isConsideredOlderThan';
 const PlayAgeRangeDeclarationHybridObject =
   NitroModules.createHybridObject<PlayAgeRangeDeclaration>(
     'PlayAgeRangeDeclaration'
   );
 
-export async function getAppleDeclaredAgeRangeStatus(
```

Comment:
I'm considering adding all the thresholds back as optional and passing them to requestDeclaredAgeRange if provided else can can use the ageRangeThresholdManager. If the Developer needs that control. But it's pretty important for the user experience that we don't pass in new thresholds every request. If we do the result is not cached